### ISSUE TITLE
[v7.1] Gantt View Grey Screen

### ISSUE THREAD

**Issue by applepipe** [frappe/erpnext#6499]
Hello,

I'm testing the new Gantt live in my company, but since the update today the screen becomes grey when I try to view the chart.

Here is what the chrome console shows:

```
localStorage cleared
desk.min.js:9 Cleared App Cache.
desk.min.js:12 localStorage cleared
chrome-extension://jjoehgfmpefldljiipnmgnfmcbfjkaad/certisignerconf.js:4 A div certisigner-enabled não foi encontrada.
list.min.js:131 Uncaught ReferenceError: Gantt is not defined(anonymous function) @ list.min.js:131eval_assets @ desk.min.js:15(anonymous function) @ desk.min.js:14callback @ desk.min.js:17callback @ desk.min.js:130200 @ desk.min.js:132(anonymous function) @ desk.min.js:137i @ jquery.min.js:2fireWith @ jquery.min.js:2z @ jquery.min.js:4(anonymous function) @ jquery.min.js:4
```

Also, I think that the link for Gantt view should be removed from the page below:

![image](https://cloud.githubusercontent.com/assets/18335894/18953354/704b2a3c-8624-11e6-8ea9-e8070ec27099.png)

**Comment by netchampfaris** [comment]
Try `bench update`

**Comment by applepipe** [comment]
Same result after a bench update.

![image](https://cloud.githubusercontent.com/assets/18335894/18955924/cb783930-862f-11e6-91f1-a381280d8a2e.png)

**Comment by applepipe** [comment]
After migration it gives me this error:

Wrote ./assets/css/frappe-web.css - 54k
--Error in:/home/frappe/frappe-bench/apps/frappe/frappe/public/js/frappe/views/gantt.js--
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/build.py", line 130, in pack
    jsm.minify(tmpin, tmpout)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/minify.py", line 210, in minify
    self._jsmin()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/minify.py", line 180, in _jsmin
    self._action(3)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/minify.py", line 156, in _action
    raise UnterminatedRegularExpression()
UnterminatedRegularExpression

**Comment by netchampfaris** [comment]
Will be fixed with frappe/frappe#2127

**Comment by applepipe** [comment]
thank you!
